student.rb

```diff
@@ -1,6 +1,9 @@
 require_relative 'person'
+require_relative 'classroom'
 
 class Studen < Person
+  attr_reader :classroom
+
   def initializate(age, classroom, name = 'Unknown', parent_permission: true)
     super(name, age, parent_permission)
     @classroom = classroom
```

Comment:
Kindly correct the method name **initializate** to initialize.
However, if things have change, kindly let me know.

Reply:
Well resolved 👍🏽